src/filter.js

```diff
@@ -1,9 +1,11 @@
-import {customElement, bindable, bindingMode} from 'aurelia-framework';
+import {inject, customElement, bindable, bindingMode} from 'aurelia-framework';
 import {resolvedView}                         from 'aurelia-view-manager';
 import {CriteriaBuilder}                      from './criteriaBuilder';
+import {Configuration}                        from 'aurelia-config';
 
 @customElement('filter')
 @resolvedView('spoonx/filter', 'filter')
+@inject(Configuration.of('aurelia-filter'))
```

Comment:
I thought `aurelia-config` was part of `spoonx-tools`, just checked, and it is not the case... 

Then I guess we have the same issue for `aurelia-form`.